gomock/matchers.go

```diff
@@ -366,3 +387,14 @@ func AssignableToTypeOf(x any) Matcher {
 func InAnyOrder(x any) Matcher {
 	return inAnyOrderMatcher{x}
 }
+
+// OneOf is a Matcher that returns true when the parameter of the mock is
+// part of the set provided to the matcher
+//
+// Example Usage:
+//
+//	OneOf([]any{100,200,300})
+//	OneOf([]any{"Go", "Gopher"})
+func OneOf(x []any) Matcher {
```

Comment:
Maybe we can use .`..` instead of slice?
`func OneOf(x any...) Matcher`

Reply:
@Tulzke I personally agree, and I did that in my PR #63. 😉